### PR TITLE
Artifacts

### PR DESCRIPTION
**Pull request by BugFreeSoftware**
Test removal of schema-tool-generated artifacts

### CODE REVIEW

**Comment by dessaya** [comment]
:heart:

File: Makefile [hunk 1]
```diff
@@ -33,8 +33,8 @@ test-full: install
 test: install
 	go test -tags $(BUILD_TAGS) $(TEST_PKG) --timeout 40m --count 1 -failfast $(TEST_ARG)
 
-test-short:
-	go test -tags $(BUILD_TAGS) --short --count 1 -failfast ./...
+test-short-no-wasm:
```
Comment (dessaya) [inline Makefile]: The `test-short` rule is mentioned in the documentation. I suggest doing a global search & replace to update it. Or maybe better, don't change the name?

Comment (BugFreeSoftware) [reply Makefile]: This is only a (failing) try-out for now. I agree that changing the name is not necessary when the original test-short is no longer there. I reported my findings to Yanghao so he can try to fix things.

**Review by jorgemmsilva — APPROVED** [review]
feels-good-man

**Pull request merged**